pkg/controller/instance/instance_controller.go

```diff
@@ -164,7 +175,7 @@ func add(mgr manager.Manager, r reconcile.Reconciler) error {
 					if current.Status.State == kudov1alpha1.PhaseStateComplete {
 						log.Println("InstanceController: Current Plan for Instance is already done, won't change the Suspend flag.")
 					} else {
-						log.Println("InstanceController: Setting PlanExecution to Suspend")
+						log.Println("InstanceController: Suspending the PlanExecution")
```

Comment:
```suggestion
						log.Println("InstanceController: Suspending the PlanExecution for instance %v", new.Name)
```

Reply:
Note: this has to be `log.Printf(...)` instead of `log.Println(...)`, I made the same mistake a few times during this refactoring 😳.